src/Host.cpp

```diff
@@ -1247,7 +1247,7 @@ bool Host::uninstallPackage(const QString& packageName, int module)
     // So the package itself can eliminate what you need without having to
     // reopen the session.
     TEvent uninstallEvent;
-    uninstallEvent.mArgumentList.append( tr( "sysUninstallPackageEvent", "This string may not want/need to be translated" ) );
+    uninstallEvent.mArgumentList.append( QStringLiteral( "sysUninstallPackageEvent" ) );
```

Comment:
Yeah, I'm just reading up on 'em.  One thing though, if you want to include some variable replacements `QLatin1String` does not have `arg(...)` methods - so you'd have to assemble such `QStrings` piece-by-piece but the `%` concatenation operator is superior to the `+` one (may need to `#include <QStringBuilder>`) apparently! :open_mouth: